test/units/executor/test_action_write_locks.py

```diff
@@ -0,0 +1,95 @@
+# (c) 2021, Kevin Carter <kecarter@redhat.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+import sys
+import unittest
```

Comment:
New unit test files should use `pytest` style tests instead of `unittest`.

Reply:
I didn't see `pytest` used anywhere within the executor tests. Is there a doc I should follow when implementing these tests?

Reply:
There are many examples through the unit test dir.  I see that `test/units/executor/module_common/test_modify_module.py` uses pytest style tests.

Reply:
Looking at the docs, `pytest` will automatically pickup unittest uses, however, should I port this over to using base fixtures instead?

Reply:
Seems there's a fairly even distribution of both `unittest` and `pytest`, throughout the test directories (with no particular style implementation) is there some documentation I can follow to rewrite these tests?

Reply:
I've rewritten the tests following a simple pytest style guide. Please rereview whenever you have a moment and let me know if I should make further adjustments.